cdap-ui/app/cdap/components/AbstractWidget/SchemaEditor/RowButtons/FieldAttributes/RecordEnumAttributes.tsx

```diff
@@ -68,9 +69,11 @@ function RecordEnumTypeAttributes({
           }}
         />
       </div>
-      <Button variant="contained" color="primary" onClick={onChangeHandler}>
-        Save
-      </Button>
+      <If condition={typeof onChange === 'function'}>
```

Comment:
if there is no onChange, do we need to show this drop down?

Reply:
This is when displaying in pipeline detailed view. Since we can't edit this will hide the Save button.

Reply:
The logic to determine whether to render the component (based on the type) resides in `RowButton`, the parent container.